EpsilonLights/Src/Lights.c

```diff
@@ -49,24 +49,15 @@ void updateLightsTask(void const* arg)
         /* UPDATE BRAKE LIGHTS */
         brakes = (driversInputs[BRAKES_INPUT_INDEX_P1] >> BRAKES_INPUT_INDEX_P2) & 1;
 
-        if (brakes)
+        if (brakes || regenInputs[SET_CURRENT_INDEX] == 0 && regenInputs[SET_VELOCITY_INDEX] != 0)
```

Comment:
Either the logic or the labels are mismatched - regen braking is when the velocity is set to zero, and the current is set to something